### PR TITLE
Explicitly state our log technical limits

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Explain the limitation that are in place for logs.

### Motivation
Limitations should not be hidden, it should be clearly stated.

### Additional Notes
Keep in mind that those limits can be modified if requested.

### CODE REVIEW

File: content/logs/processing.md [hunk 1]
```diff
@@ -159,6 +159,30 @@ These settings:
 Give the following results:
 {{< img src="logs/processing/useragent_processor.png" alt="Useragent processor" responsive="true" popup="true">}}
 
+## Technical limits
+
+To make sure the Log Management solution functions in an optimal way we set the following technical limits and rules to your log events as well as to some product features. These have been designed so you never reach them. However, if this is the case please read the actions we enforce and please contact us if you have any suggestion.
```
Comment (tmichelet) [inline content/logs/processing.md]: `so you never reach them` -> `so you may never reach them`?

Comment (tmichelet) [reply content/logs/processing.md]: also double `please`

Comment (tmichelet) [reply content/logs/processing.md]: Not sure about `However, if this is the case please read the actions we enforce and please contact us if you have any suggestion`, I'd just remove this line

File: content/logs/processing.md [hunk 2]
```diff
@@ -159,6 +159,30 @@ These settings:
...
+
+### Limits applied to ingested log events
+
+* The size of a log event should not exceed 25K characters.
```
Comment (tmichelet) [inline content/logs/processing.md]: characters -> bytes

File: content/logs/processing.md [hunk 3]
```diff
@@ -159,6 +159,30 @@ These settings:
...
+* Log events can be submitted up to 6h in the past and 2h in the future into an index.
```
Comment (tmichelet) [inline content/logs/processing.md]: `into and index` is too technical, let's remove those words

File: content/logs/processing.md [hunk 4]
```diff
@@ -159,6 +159,30 @@ These settings:
...
+* A log event once converted to JSON format should contain less than 256 attributes, each of those attribute’s key should be less than 50 characters, be nested in less than 10 successive levels and their respective value be less than 1024 characters if promoted as a facet.
+* A log event should not have more than 100 tags and each tag key should not exceed 50 characters and their respective value should not exceed 100 characters for a maximum of 10 millions unique tag per day.
```
Comment (tmichelet) [inline content/logs/processing.md]: could we say that tags should not exceed `256 chars` instead? no need to separate key and value

**Comment by NBParis** [comment]
@tmichelet Updated with your remarks

File: content/logs/processing.md [hunk 5]
```diff
@@ -159,6 +159,29 @@ These settings:
...
+To make sure the Log Management solution functions in an optimal way we set the following technical limits and rules to your log events as well as to some product features. These have been designed so you may never reach them.
+
+### Limits applied to ingested log events
+
+* The size of a log event should not exceed 25K bytes.
+* Log events can be submitted up to 6h in the past and 2h in the future.
+* A log event once converted to JSON format should contain less than 256 attributes, each of those attribute’s key should be less than 50 characters, be nested in less than 10 successive levels and their respective value be less than 1024 characters if promoted as a facet.
+* A log event should not have more than 100 tags and each tag should not exceed 256 characters for a maximum of 10 millions unique tag per day.
+
+Log events which enforce these limits might be transformed or truncated by the system. Or simply not indexed if outside of the provided time range. However, be sure that Datadog always tries to do its best to preserve as much as possible the provided user data.
```
Comment (egueidan) [inline content/logs/processing.md]: enforce -> do not comply/fall out of/...

File: content/logs/processing.md [hunk 6]
```diff
@@ -97,7 +97,7 @@ If your logs put their dates in an attribute not in this list, use the log date
 {{< img src="logs/processing/log_date_remapper.png" alt="Log date Remapper" responsive="true" popup="true">}}
 
 If your logs don't contain any of the default attributes and you haven't defined your own date attribute, Datadog timestamps the logs with the date it received them.  
-If the log's official timestamp is from one of the default attributes or [an attribute of your choosing](/logs/processing/#log-date-remapper), Datadog rejects the log if the date is more than 18 hours in the past.
+If the log's official timestamp is from one of the default attributes or [an attribute of your choosing](/logs/processing/#log-date-remapper).
```
Comment (egueidan) [inline content/logs/processing.md]: Is that a sentence?

Comment (NBParis) [reply content/logs/processing.md]: @egueidan changed.

File: content/logs/processing.md (hunk 5 above)
Comment (NBParis) [reply content/logs/processing.md]: @egueidan updated

**Pull request merged**